src/common/components/hosts/HostStatus.tsx

```diff
@@ -117,7 +117,9 @@ const withProgress = (
     'error',
     'cancelled',
   ].includes(status || '');
-  return shouldAppendProgress ? `${title} ${currentStage}/${totalStages}` : title;
+  return shouldAppendProgress && totalStages > 0
```

Comment:
`shouldAppendProgress` variable should carry the information about whether to show or not show the progress.